scenarios/command/README.md

```diff
@@ -144,11 +144,13 @@ host_name=$(az resource show --ids $res_id --query "properties.topicSpacesConfig
 
 echo "MQTT_HOST_NAME=$host_name" > vehicle03.env
 echo "MQTT_USERNAME=vehicle03" >> vehicle03.env
+echo "MQTT_CLIENT_ID=vehicle03" >> vehicle03.env
```

Comment:
with the latest MQTTnet there was a runtime error when trying to connect without clientID.

@vaavva do you have any issue in C without the clientID ?

Reply:
Yes, we need the client ID - I had added it in this PR, not sure when/why it got removed:
https://github.com/Azure-Samples/MqttApplicationSamples/pull/50/files#diff-43217893e8ab09f14b99ecf046bd0d51da718e001352611576786ebc2d8ac007

Reply:
seems I removed it in #57 :(  .. my bad, wondering how we did not catch the issue..